pkg/drain/evictor/filters.go

```diff
@@ -158,7 +158,7 @@ func hasLocalStorage(pod corev1.Pod) bool {
 	return false
 }
 
-func (d *Helper) annotationFilter(pod corev1.Pod) podDeleteStatus {
+func (d Helper) annotationFilter(pod corev1.Pod) podDeleteStatus {
```

Comment:
What's the most idiomatic way to do this? I know only that most of the code uses pointers and using value here I think might cause unexpected bugs if someone decides to mutate the passed object later on?

Reply:
Good spot, changing to a pointer